node-support/bootloader/src/gas_wallet.rs

```diff
@@ -0,0 +1,119 @@
+//! Helpers for setting up a gas wallet for the relayer
+
+use common::types::hmac::HmacKey;
+use external_api::auth::add_expiring_auth_to_headers;
+use reqwest::{
+    header::{HeaderMap, HeaderValue, CONTENT_TYPE},
+    Client,
+};
+use serde::{Deserialize, Serialize};
+use std::{collections::HashMap, time::Duration};
+
+use libp2p::PeerId;
+use toml::Value;
+use tracing::{info, warn};
+
+use crate::helpers::read_env_var;
+
+// -------------
+// | Constants |
+// -------------
+
+/// The chain env environment variable
+///
+/// This is a value like `arbitrum-sepolia`
+const ENV_CHAIN_ENV: &str = "CHAIN_ENV";
```

Comment:
we don't need this as a new env var, the `CHAIN` env var will be exactly this value and is already passed in to the relayer container